### DISCUSSION TITLE
Is giters.com owned by github?

### DISCUSSION THREAD

**Discussion by Dfmaaa** [community/community#22428] [New to GitHub]
<p>So, I searched for some software, I saw a website looking exactly like github, it’s name is <a href="http://giters.com" rel="noopener nofollow ugc">giters.com</a>, I am really confused, is it a scam or is it a part of github?</p>

**Comment by ossanna16** [comment, marked as answer]
<p>Hi there! This site is not owned by GitHub.</p>

**Reply by mcdiamondz** [reply]
if it is not owned by Github why am I seeing all my repositories including the repos I forked in there?

**Comment by Cardosofiles** [comment]
Giters.com is not officially part of GitHub, so it's essential to exercise caution when browsing or using sites that imitate well-known platforms. Although Giters.com appears to offer similar functionalities to GitHub, it’s a third-party site that mirrors public GitHub repositories, aggregating information and enabling search and viewing of projects hosted on GitHub. However, it has no direct affiliation with or official support from GitHub.

Here are some important precautions:

Login and Credentials: Avoid logging into sites like Giters.com with your GitHub credentials. Since it’s not an official site, it may pose a security risk.

Sensitive Data: Never input personal information, access tokens, or other sensitive data on a site that is not directly GitHub (github.com).

Official Links: To access GitHub and all its associated services, always use links from the official github.com domain.

If you want to verify the authenticity of sites related to GitHub, you can consult the [Official GitHub Documentation](https://docs.github.com/) or [GitHub Status](https://www.githubstatus.com/) for information on authorized resources.

**Comment by seefeldmaxwell** [comment]
Same here no reason why alot of my old code is on there crazy
